zerver/lib/push_notifications.py

```diff
@@ -1200,6 +1220,14 @@ def handle_push_notification(user_profile_id: int, missed_message: Dict[str, Any
     send_apple_push_notification(user_identity, apple_devices, apns_payload)
     send_android_push_notification(user_identity, android_devices, gcm_payload, gcm_options)
 
+    do_increment_logging_stat(
+        user_profile.realm,
+        COUNT_STATS["mobile_pushes_sent::day"],
+        None,
+        timezone_now(),
+        increment=len(android_devices) + len(apple_devices),
+    )
```

Comment:
This runs on the bouncer itself, tracking the stats for notifications sent by zulipchat to our own users - it felt natural to have this stat tracked here too, not only on self-hosted server. Unless performance or other concerns make us not want this on the bouncer? If so, we can drop it, since this isn't truly needed.

Reply:
Yeah, this seems correct.